### PR TITLE
Fix MyPy Errors for SSH provider

### PR DESCRIPTION
**Pull request by kazanzhy**
Fix MyPy Errors for SSH provider

### CODE REVIEW

File: airflow/providers/ssh/hooks/ssh.py [hunk 1]
```diff
@@ -305,7 +305,7 @@ def get_conn(self) -> paramiko.SSHClient:
         client.connect(**connect_kwargs)
 
         if self.keepalive_interval:
-            client.get_transport().set_keepalive(self.keepalive_interval)
+            client.get_transport().set_keepalive(self.keepalive_interval)  # type: ignore[union-attr]
```
Comment (kazanzhy) [inline airflow/providers/ssh/hooks/ssh.py:310]: I have to use "ignore" here because of "paramiko" lib isn't well typed.
Maybe someone has any suggestions?

Comment (potiuk) [reply airflow/providers/ssh/hooks/ssh.py:310]: I think just add the comment above it why the ignore. This is only a local problem that will nt bubble to any users of the provider.

File: airflow/providers/ssh/operators/ssh.py [hunk 2]
```diff
@@ -206,7 +206,7 @@ def run_ssh_client_command(self, ssh_client: SSHClient, command: str) -> bytes:
         return agg_stdout
 
     def execute(self, context=None) -> Union[bytes, str]:
-        result = None
+        result: Optional[Union[bytes, str]] = None
```
Comment (uranusjr) [inline airflow/providers/ssh/operators/ssh.py:209]: I think you can simply delete this line.

Comment (uranusjr) [inline airflow/providers/ssh/operators/ssh.py]:
```suggestion
        result: Union[bytes, str]
```

**Review by uranusjr — APPROVED** [review]

File: airflow/providers/ssh/hooks/ssh.py (hunk 1 above)
Comment (kazanzhy) [reply airflow/providers/ssh/hooks/ssh.py:310]: Added

**Review by potiuk — APPROVED** [review]

**Pull request merged**